local/nginx/include/upstream.conf

```diff
@@ -0,0 +1,31 @@
+upstream content-server {
+    server content-server:6969;
+}
+
+upstream comms-server {
+    server comms-server:9000;
+}
+
+upstream lambdas {
+    server lambdas:7070;
+}
+
+upstream comms_metrics {
+    server comms-server:9090;
+}
+
+upstream content_metrics {
+    server content-server:9090;
+}
+
+upstream lambdas_metrics {
+    server lambdas:7070;
```

Comment:
Just wondering, is there a reason why the other metrics are on port `9090` and these are in `7070`?

Reply:
It just works. There is a different default in the lambdas project